datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/DurationSerializer.java

```diff
@@ -135,4 +183,9 @@ protected JsonToken serializationShape(SerializerProvider provider) {
     protected JSR310FormattedSerializerBase<?> withFeatures(Boolean writeZoneId, Boolean writeNanoseconds) {
         return new DurationSerializer(this, _useTimestamp, writeNanoseconds, _formatter);
     }
+
+    @Override
+    protected DateTimeFormatter _useDateTimeFormatter(SerializerProvider prov, JsonFormat.Value format) {
+        return null;
```

Comment:
This is the part I was stuck on a bit, suddenly thinking the DateTimeFormatter could be put to use, and then after spinning on that a bit, found I agree with whoever added this comment (line 103 of original `DurationSerializer`.
 // Does not look like we can make any use of DateTimeFormatter here?

Reply:
This is because serializer is extending `JSR310FormattedSerializerBase` which has a common `DateTimeFormatter`. But `Duration` is not supported by `DateTimeFormatter`, actually it has its own formatter embedded in the class based on ISO-8601. I removed the comment because once this is clear, does not make sense to use the formatter of the base class in the `DurationSerializer`.

For the long term, I think `JSR310FormattedSerializerBase` can be refactored a bit extracting the common functionality of classes extending `DateTime` into its own base (maybe `JSR310FormattedSerializerDateTimeBase`), so there is no need to override `_useDateTimeFormatter` in this case.

Hope this clarifies a bit my point of view.